src/pages/ahnurim/Main/Main.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable */
+import React, { useEffect } from 'react';
+import { useState } from 'react';
+import '../Main/Main.scss';
```

Comment:
```suggestion
import React, { useEffect, useState } from 'react';
import '../Main/Main.scss';
```

useState도 React의 내장 hook 입니다.
함께 import 해주세요